docs/wallets/wallets.mdx

```diff
@@ -31,9 +31,6 @@ Generally, this wallet focuses on people who want to send or receive HNT or
 manage hotspots. Other activities, such as managing end-devices and OUIs (i.e.
 users of the network) are generally best suited for the command line wallet.
 
-To learn more about the wallet, please visit
-[our help website here](https://intercom.help/heliumnetwork/en/articles/3287804-app-overview).
-
```

Comment:
Thanks for catching this. We actually just migrated these docs over to docs.helium. Can you update this and drop this link in:

https://docs.helium.com/wallets/app-wallet

It would actually just need to be `/wallets/app-wallet`